web/ja/manual/function/generator/intro/alterbyintro.html

```diff
@@ -1,53 +1,303 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
-	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="ja" lang="ja" >
 <head>
 	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
 	<meta http-equiv="Content-Style-Type" content="text/css" />
 	<meta name="keywords" content="DBFlute,DBFlute Intro,イントロ,支援ツール,DB変更,AlterCheck" />
-	<style type="text/css" title="currentStyle">@import "../../../../../css/sub.css";</style>
+	<link rel="stylesheet" type="text/css" href="../../../../../css/sub.css" />
+	<!--[if IE 6.0]>
+	<link rel="stylesheet" type="text/css" href="../../../../../css/cb/sub.css" />
+	<![endif]-->
 	<title>AlterCheck を Intro で | DBFlute</title>
 </head>
 <body>
 <div id="container" class="ct-manual">
-<div id="mainbar"><div id="header">
-	<a href="#">DBFlute<span>DB変更に強いO/Rマッパー</span></a>
-	<img src="../../../../../image/logo-content.gif" alt="logo"/>
-	<ul>
-		<li class="hd-introduction"><a href="#">Introduction</a></li>
-		<li class="hd-tutorial"><a href="#">Tutorial</a></li>
-		<li><ul><li class="hd-architect"><a href="#">for Architect</a></li><li class="hd-developer"><a href="#">for Developer</a></li></ul></li>
-		<li class="hd-environment"><a href="#">Environment</a></li>
-		<li><ul><li class="hd-install"><a href="#">Install</a></li><li class="hd-upgrade"><a href="#">Upgrade</a></li></ul></li>
-		<li class="hd-manual"><a href="#">Manual</a></li>
-	</ul>
-</div>
 
-<div id="content"><!-- __content-start__ -->

```

Comment:
@EgumaYuto  Please restore dummy header for Groovy script of auto-header.

For example, the <div id="content"><!-- __content-start__ --> is important mark for the script.

Reply:
@jflute 
I fixed it in this commit.
https://github.com/dbflute/dbflute-document/pull/4/commits/a94d8e6e5c8b75d87a1026ecd07b11d8cbd6895d